prac_06/guitar.py

```diff
@@ -0,0 +1,20 @@
+class Guitar:
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : {self.cost}"
+
+    def get_age(self):
+        age = 2023 - self.year
+        return age
+
+    def is_vintage(self):
+        age = self.get_age()
+        if age >= 50:
+            return True
+        else:
+            return False
```

Comment:
if the usage of get_age is only to determine if the guitar is vintage, you could combine it into 1 function or even check for it within the constructor. I wouldn't consider this method more elegant, therefore your solution is still on par.